juju/juju
# provider/azure: import credentials from Azure CLI

## Description of change

Backport of https://github.com/juju/juju/pull/7457 to 2.2.

## QA steps

See https://github.com/juju/juju/pull/7457

Additionally I ran through the existing add-credential flow to verify that still works.

## Documentation changes

See https://github.com/juju/juju/pull/7457

## Bug reference

See https://github.com/juju/juju/pull/7457
